tqp/topic.py

```diff
@@ -7,8 +7,17 @@
 
 class Topic:
     def __init__(self, topic_name):
+        self.topic_name = topic_name
+        self._topic = None
+
+    @property
+    def topic(self):
```

Comment:
eh? do we sometimes not access the topic?

could just use http://werkzeug.pocoo.org/docs/0.14/utils/#werkzeug.utils.cached_property